config/locales/en.yml

```diff
@@ -234,7 +234,7 @@ en:
     resend_confirmation: Resend confirmation instructions
     reset_password: Reset password
     set_new_password: Set new password
-    invalid_reset_password_token: Password reset link is invalid or expired. Please try again.
+    invalid_reset_password_token: Password reset link is invalid or expired. Have you already used it?
```

Comment:
As per https://github.com/tootsuite/mastodon/blob/8d6c3cd48ae4f96752ff6b698bc7244d97aa9a27/config/initializers/devise.rb#L230, password reset token expires in 6 hours without consume it. I think "Please request a new one." is better.